src/components/search-bar/story-header.dev.inc.tsx

```diff
@@ -23,6 +37,19 @@ const StoryHeader: React.FC<StoryHeaderProps> = ({ search, userProfile }) => {
   // replaced by {children}
   const [isEnabledAdvancedSearch, setIsEnabledAdvancedSearch] = useState(true);
   const headerMenuRef = React.useRef<HTMLDivElement>(null);
+  const t = useText();
+
+  useEffect(() => {
+    if (window.googleTranslateElementInit) return;
```

Comment:
You must check the external library feature `window.google.translate.TranslateElement`, not your init function.